<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchFrame</class>
 <widget class="AccordionFrame" name="SearchFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="searchInComboBox">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search the Info column of the packet list (summary pane), decoded packet display labels (tree view pane) or the ASCII-converted packet data (hex view pane).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>Packet list</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Packet details</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Packet bytes</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QComboBox" name="charEncodingComboBox">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search for strings containing narrow (UTF-8 and ASCII) or wide (UTF-16) characters.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>Narrow &amp; Wide</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Narrow (UTF-8 / ASCII)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wide (UTF-16)</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="caseCheckBox">
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QComboBox" name="searchTypeComboBox">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search for data using display filter syntax (e.g. ip.addr==10.1.1.1), a hexadecimal string (e.g. fffffda5) or a plain string (e.g. My String).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>Display filter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hex value</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>String</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="SyntaxLineEdit" name="searchLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="findButton">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccordionFrame</class>
   <extends>QFrame</extends>
   <header>accordion_frame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SyntaxLineEdit</class>
   <extends>QLineEdit</extends>
   <header>syntax_line_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
